src/bufr/BufrParser/Exports/Export.cpp

```diff
@@ -35,6 +36,10 @@ namespace
             const char* Datetime = "datetime";
             const char* Query = "query";
             const char* GroupByField = "group_by";  // Deprecated
+<<<<<<< HEAD
```

Comment:
@nicholasesposito there are conflicts here that are not resolved...

Reply:
Got ahead of myself. Stand by...